test/unit-luatest/storage_exports_test.lua

```diff
@@ -135,3 +135,16 @@ test_group.test_deploy_privs = function(g)
         box.schema.role.drop('two')
     end)
 end
+
+test_group.test_core_version = function(g)
+    g.server:exec(function()
+        local vexports = require('vshard.storage.exports')
```

Comment:
You can use `ivexports`, it is defined for this suite in `test_group.before_all`.

Reply:
Sure, didn't notice it. Fixed